apps/spruce/src/gql/queries/version-tasks.graphql

```diff
@@ -10,9 +10,10 @@ query VersionTasks(
       data {
         aborted
         baseTask {
+          displayStatus
           execution
           id
-          status
+          status: displayStatus
```

Comment:
I guess I don't fully understand why we need to alias `status` as `displayStatus`. Could we just keep `status` and use the new field `displayStatus` as we prepare for the transition of these fields? I assume that this will require multiple PRs anyways to migrate the field

Reply:
I initially kept both statuses as a backup, just in case I missed something during the refactor and didn’t want the page to break. But I decided to go ahead and update everything, and wow, there are so many places that rely on `status`. TypeScript wasn’t too happy with that, so the PR ended up growing more than I expected